app/models/unit.rb

```diff
@@ -484,7 +465,7 @@ def enrol_student(user, campus)
     Project.create!(
       user_id: user.id,
       unit_id: id,
-      task_stats: '0.0|1.0|0.0|0.0|0.0',
+      task_stats: '0.0|1.0|0.0|0.0|0.0|0.0',
```

Comment:
**question:** Are we supposed to set this as a JSON string? 🤔 

```rb
{               
  red_pct: 0,   
  grey_pct: 1,  
  orange_pct: 0,
  blue_pct: 0,  
  green_pct: 0, 
  order_scale: 0
}               
```

Reply:
🤦

Reply:
Absolutely :)